src/main/java/net/mcreator/blockly/BlocklyToCode.java

```diff
@@ -128,6 +129,20 @@ public final void processBlockProcedure(List<Element> blocks) throws TemplateGen
 		for (Element block : blocks) {
 			String type = block.getAttribute("type");
 			boolean generated = false;
+			for (StatementInput statementInput : this.statementInputStack) {

```

Comment:
I would move this code to 'GetVariableBlock' and 'SetVariableBlock' classes, so this processing is only done when a local variable is used, and there use 'BlocklyToCode master' to determine if statement disabling local variables is used from 'statementInputStack'. This will also remove the need for 'getAllChildrenWithName' which will greatly improve the performance of this.